styles.css

```diff
@@ -130,6 +152,29 @@ p {
   font-size: 4vw;
 }
 
+.sponsorship-info {
+  margin-bottom: 3em;
+  padding-left: 1.5em;
+  padding-right: 1.5em;
+}
+
+.sponsorship-info section {
+  background-color: rgba(250, 250, 250, 0.5);
+  padding: 1em 1em 1em 1em;
+  margin-bottom: 1em;
+  border-radius: 0.5em;
+}
+
+.vip-sponsors__made-tech-image {
+  height: 25%;
+  width: 50%;
+}
+
+.previous-sponsors__image {
+  height: 10%;
+  width: 20%;
+}
+
 /* desktop */
 @media screen and (min-width: 1025px) {
```

Comment:
minor/styling:
Please could we add a `max-width: 60vw` to the `sponsorship-info` class in the desktop @media section, it helps with readability on big screens 🖥️